orchagent/portsorch.cpp

```diff
@@ -109,8 +110,8 @@ PortsOrch::PortsOrch(DBConnector *db, vector<string> tableNames) :
     /* Set port to hardware learn mode */
     for (i = 0; i < m_portCount; i++)
     {
-        attr.id = SAI_PORT_ATTR_FDB_LEARNING;
-        attr.value.s32 = SAI_PORT_LEARN_MODE_HW;
+        attr.id = SAI_BRIDGE_PORT_ATTR_FDB_LEARNING_MODE;
+        attr.value.s32 = SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW;
```

Comment:
FDB learning mode is a bridge port attribute and not a port attribute
Here, set_port_attribute is used, which seems incorrect
Need to add a flow that after init : 1) gets default .1q bridge 2) read all bridge ports list 3) per bridge port reads the port ID 4) thus maintain a map between a port and bridge port
And then here to call the API on the relevant bridge port